README.md

```diff
@@ -4,39 +4,49 @@
 
 ## Contents
 
-- `library`: Ansible modules files for KubeVirt management
-- `tests`: Ansible playbooks to test KubeVirt modules
+- `lib`: Ansible modules files for KubeVirt management
+- `tests`: Ansible playbook examples and unit tests
 
 ## Requirements
 
-- Ansible 2.4.3.0
-- Kubernetes Python Module
+- Ansible >= 2.4.3.0
+- [KubeVirt Python SDK](https://github.com/kubevirt/client-python)
 - [KubeVirt](https://github.com/kubevirt/kubevirt)
 
 ## Installation and usage
 
-Use the Galaxy client to install the role:
+* Using Git
 
 ```
-$ ansible-galaxy install kubevirt.kubevirt-modules
+$ git clone https://github.com/kubevirt/ansible-kubevirt-modules
 ```
 
-Once installed, add it to a playbook:
+* [Install KubeVirt Python SDK](https://github.com/kubevirt/client-python#installation--usage)
+
+* Once installed, add it to a playbook:
 
 ```
 ---
 - hosts: localhost
   remote_user: root
   roles:
-    - role: kubevirt.kubevirt-modules
+    - role: ansible-kubevirt-modules
       install_python_requirements: no
     - role: hello-underworld
 ```
-> **NOTE:** Set `install_python_reqirememnts: yes` to install Kubernetes Python module
 
 Because the role is referenced, the `hello-underworld` role is able to make use of the kubevirt modules
 
-## LOCAL TESTING
+## Playbook examples
+
+* [Virtual Machine](tests/raw_vm.yml)
+* [Offline Virtual Machine](tests/raw_ovm.yml)
+* [Virtual Machine ReplicaSet](tests/raw_vmrs.yml)
+* [Virtual Machine facts](tests/kubevirt_vm_facts.yml)
+* [Offline Virtual Machine facts](tests/kubevirt_ovm_facts.yml)
+* [Virtual Machine Replica Set facts](tests/kubevirt_vmrs_facts.yml)
+
+## Local testing
```

Comment:
I would change it into a numerated list too. It's easier to follow.

Reply:
@tripledes Sergi, sorry it wasn't clear maybe. This comment about a numerated list was about instructions in Local Testing section, not about list of playbooks examples. I can merge the PR now, it's not that important, and it can be updated later, agree?

Reply:
@alexxa no worries, the small GH preview mislead me :smiley: , I just pushed a new update which should make it better.